spring-boot-samples/spring-boot-sample-data-geode/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<!-- Your own application should inherit from spring-boot-starter-parent -->
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-samples</artifactId>
+		<version>1.4.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-boot-sample-data-geode</artifactId>
+	<name>Spring Boot Data Geode Sample</name>
+	<description>Spring Boot Data Geode Sample</description>
+	<url>http://projects.spring.io/spring-boot/</url>
+	<organization>
+		<name>Pivotal Software, Inc.</name>
+		<url>http://www.spring.io</url>
+	</organization>
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+		<spring-shell.version>1.1.0.RELEASE</spring-shell.version>
```

Comment:
1. I could not find the _Spring Shell_ **version** declared anywhere in _Spring Boot_ (e.g. `spring-boot-dependencies/pom.xml`).

2. There is no `spring-boot-starter-shell`, or the like for _Spring Shell_ in `[spring-boot-starters](https://github.com/jxblum/spring-boot/tree/master/spring-boot-starters)` (AFAICT).

3. The `spring-boot-starter-data-gemfire/geode` starters do not declare the `spring-shell` dependency directly, since _Spring Shell_ is not a hard, required dependency.  It is only needed by GemFire/Geode if a Server is also to become a "Manager" (managing node in the cluster, which it is not be default and so needs to be explicitly enabled).  However, that does not prevent GemFire or Geode from erroneously trying to load the GemFire shell "command" classes, which are also used by the Manager within a cluster to carry out the remote, or cluster-based commands.  These command classes implement _Spring Shell_ classes/interfaces.  If they are not found, then GemFire/Geode throws a rather annoying (noisy), but benign Exception.  You can see this on GemFire/Geode startup if you remove the "explicit" "runtime-only" _Spring Shell_ dependency.

So, technically, the `spring-shell.version` is not been overridden (AFAIK).  It is also unnecessary but was put there to suppress the Exception stack trace in the log output, which is meaningless and confusing in this sample application context.